src/source/PeerConnection/PeerConnection.c

```diff
@@ -477,32 +477,37 @@ STATUS generateJSONSafeString(PCHAR pDst, UINT32 len)
 STATUS rtcpReportsCallback(UINT32 timerId, UINT64 currentTime, UINT64 customData)
 {
     STATUS retStatus = STATUS_SUCCESS;
+    BOOL ready       = FALSE;
+    UINT64 ntpTime, rtpTime, delay;
+    UINT32 packetCount, octetCount, packetLen, allocSize, ssrc;
+    PBYTE rawPacket                       = NULL;
+    PKvsPeerConnection pKvsPeerConnection = NULL;
+
     PKvsRtpTransceiver pKvsRtpTransceiver = (PKvsRtpTransceiver) customData;
     CHK(pKvsRtpTransceiver != NULL && pKvsRtpTransceiver->pJitterBuffer != NULL && pKvsRtpTransceiver->pKvsPeerConnection != NULL, STATUS_NULL_ARG);
-    PKvsPeerConnection pKvsPeerConnection = pKvsRtpTransceiver->pKvsPeerConnection;
-    PBYTE rawPacket = NULL;
+    pKvsPeerConnection = pKvsRtpTransceiver->pKvsPeerConnection;
 
-    UINT32 ssrc = pKvsRtpTransceiver->sender.ssrc;
+    ssrc = pKvsRtpTransceiver->sender.ssrc;
     DLOGD("rtcpReportsCallback " PRIu64 " ssrc: %u rtxssrc: %u", currentTime, ssrc, pKvsRtpTransceiver->sender.rtxSsrc);
 
     // check if ice agent is connected, reschedule in 200msec if not
-    BOOL ready = pKvsPeerConnection->pSrtpSession != NULL &&
+    ready = pKvsPeerConnection->pSrtpSession != NULL &&
         currentTime - pKvsRtpTransceiver->sender.firstFrameWallClockTime >= 2500 * HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
     if (!ready) {
         DLOGD("sender report no frames sent %u", ssrc);
     } else {
         // create rtcp sender report packet
         // https://tools.ietf.org/html/rfc3550#section-6.4.1
-        UINT64 ntpTime = convertTimestampToNTP(currentTime);
-        UINT64 rtpTime = pKvsRtpTransceiver->sender.rtpTimeOffset +
+        ntpTime = convertTimestampToNTP(currentTime);
+        rtpTime = pKvsRtpTransceiver->sender.rtpTimeOffset +
             convertTimestampToRTP(pKvsRtpTransceiver->pJitterBuffer->clockRate, currentTime - pKvsRtpTransceiver->sender.firstFrameWallClockTime);
-        UINT32 packetCount = pKvsRtpTransceiver->sender.outboundRtpStreamStats.sentRtpStreamStats.packetsSent;
-        UINT32 octetCount = pKvsRtpTransceiver->sender.outboundRtpStreamStats.sentRtpStreamStats.bytesSent;
+        packetCount = pKvsRtpTransceiver->sender.outboundRtpStreamStats.sentRtpStreamStats.packetsSent;
+        octetCount  = pKvsRtpTransceiver->sender.outboundRtpStreamStats.sentRtpStreamStats.bytesSent;
         DLOGD("sender report %u " PRIu64 " " PRIu64 " : %u packets %u bytes", ssrc, ntpTime, rtpTime, packetCount, octetCount);
```

Comment:
For the formatting for PRIu64 check out: https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/ae7462f38d71823d87ed284013e2ea976b48011a/samples/Common.c#L246

Reply:
:facepalm: